### PR TITLE
feat(mj-font): add fonts option

### LINKED ISSUE #88: Document or support render options
I am not familiar enough with Rust to be certain of what I'm saying but, [if I understand correctly what I'm reading here](https://github.com/jolimail/mrml-core/blob/main/src/prelude/render.rs#L19-L22) only one render option is supported [of those [documented in MJML documentation](https://documentation.mjml.io/#inside-node-js) : `disable_comments` which seems to corresponds to the `keepComments` option in NodeJS implementation.

In particular, it doesn't seem like the `fonts` option is unsupported.

I can try to open a PR for this.

### PR DESCRIPTION
**Pull request by christophehenry**
Also add a documentation on what options are available.

Fixes #88

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -40,6 +40,14 @@ fn main() {
 }
 ```
 
+Available options are:
+
+| Name                 | Comment                                              | Default value                                                                                        |
+|----------------------|------------------------------------------------------|------------------------------------------------------------------------------------------------------|
+| `disable_comments`   | Strip comments out of rendered HTML                  | `false`                                                                                              |
+| `social_icon_origin` | Custom URL for fetching social icons                 | `None`                                                                                               |
```
Comment (christophehenry) [inline readme.md:48]: Maybe the format of the ressource at the URL should be specified here. Also I noticed that [the default URL](https://github.com/jolimail/mrml-core/blob/main/src/mj_social_element/render.rs#L10) returns an error. This probably should be checked.

**Comment by christophehenry** [comment]
@jdrouet Sorry to ping you directly but we need this feature for a project and this PR should be easy to review
.

File: readme.md (hunk 1 above)
Comment (jdrouet) [reply readme.md:48]: Thanks for spotting this, I created an issue [here](https://github.com/jolimail/mrml-core/issues/97) to track this 😉

**Comment by christophehenry** [comment]
@jdrouet Thank you very much. Would you agree to push a release?

**Comment by jdrouet** [comment]
👍 I'll push that tonight or tomorrow 😉

**Comment by christophehenry** [comment]
Awesome! Thx!

**Comment by christophehenry** [comment]
I'm sorry, I don't see `2.0.0-rc3` available on crates.io. How should I use it?

**Pull request merged**